### ISSUE TITLE
Style properties on the focussed element/elements

### ISSUE THREAD

**Issue by nikhilm19** [andrew4699/react-see-through#1]
When the style parameter of the Mask is set, the style properties don't go away after it becomes inactive

My Code: 
`        <SeeThrough 
              style={{
                "box-shadow": "5px 10px 8px 10px #888888",
              }}
              active={this.state.activeSee}
              onClick={(masked) =>
                !masked && this.setActiveSee(!this.state.activeSee)
              }
            >
`

Attaching screenshots of state active and inactive in order

<img width="1392" alt="Screenshot 2020-04-16 at 1 11 01 PM" src="https://user-images.githubusercontent.com/22270301/79428877-6a3ee800-7fe4-11ea-8d62-080e003eb251.png">
<img width="1392" alt="Screenshot 2020-04-16 at 1 11 05 PM" src="https://user-images.githubusercontent.com/22270301/79428884-6dd26f00-7fe4-11ea-8f66-58abb4730baf.png">

## Expected Behaviour :
Inactive state : 
<img width="1392" alt="Screenshot 2020-04-16 at 1 18 18 PM" src="https://user-images.githubusercontent.com/22270301/79429129-bf7af980-7fe4-11ea-9dd4-1ba10eadefa4.png">

**Comment by nikhilm19** [comment]
Updated the issue

**Comment by andrew4699** [comment]
Thank you for the feedback! I'll add this (probably sometime this week) and reply to this when it's in.

**Comment by nikhilm19** [comment]
Sounds good! But still believe you can include style for actual styling :)

**Comment by andrew4699** [comment]
I agree it would be nice to have. I'll leave this open and potentially add it in the future.
